apps/costcontrol/js/app.js

```diff
@@ -242,7 +244,7 @@ var CostControlApp = (function() {
 
   window.addEventListener('airplaneModeDisabled',
     function _onAirplanemodeDisabled(evt) {
-      if (evt.detail && evt.detail.serviceId === 'data') {
+      if (airplanemodeActive && evt.detail && evt.detail.serviceId === 'data') {
         waitForSIMReady(startApp);
       }
     });
```

Comment:
nit: please, add a breakline before the close bracket and align it below the `w` of `window.addEventListener()`